### ISSUE TITLE
groupBy symbol support

### ISSUE THREAD

**Issue by JaegerJens** [rbuckton/QueryJS#2]
I want to group my data, but my keys can have a ES6 symbol.

```js
var Query = require("iterable-query/out/es5").Query;

const foo = Symbol('foo');
const data = [
  { category: 'number', value: 'one' },
  { category: 'number', value: 'two' },
  { category: 'number', value: 'three' },
  { category: foo, value: 'foobar'}
];

const result = Query.from(data).groupBy(row => row.category).toArray();
console.log(result);
```

When I execute this program then I got this error message:
```
node_modules\iterable-query\out\es5\collections.js:277
        Object.defineProperty(target, uniqueKey, { value: key });
               ^

TypeError: Object.defineProperty called on non-object
    at Function.defineProperty (<anonymous>)
    at GetUniqueKey (\node_modules\iterable-query\out\es5\collections.js:277:16)
    at Map.get (\node_modules\iterable-query\out\es5\collections.js:47:18)
    at CreateGroupings (\node_modules\iterable-query\out\es5\query.js:5185:32)
    at GroupByIterator.next (\node_modules\iterable-query\out\es5\query.js:3796:31)
    at Object.next (\node_modules\iterable-query\out\es5\query.js:1279:35)
    at ForOf (\node_modules\iterable-query\out\es5\query.js:5091:31)
    at ToArray (\node_modules\iterable-query\out\es5\query.js:5207:5)
    at Query.toArray (\node_modules\iterable-query\out\es5\query.js:1196:16)
    at Object.<anonymous> (index.js:11:62)
```

Tested with `iterable-query@0.1.3`

**Comment by rbuckton** [comment]
I'll look into this. 

Is there a reason you are using the ES5 version if you are running in an environment that supports ES6?

**Comment by rbuckton** [comment]
This should be fixed as of `iterable-query@0.1.4`.
